src/components/paginator.js

```diff
@@ -1,16 +1,16 @@
 import { Link } from 'gatsby'
 import React from 'react'
 
-import paginatorStyles from './paginator.module.css'
+import './paginator.module.css'
```

Comment:
@raffazizzi Not exactly sure why these changes got bundled into the Gatsby3 branch? In any case, this was a CSS module file and now it looks like it's trying to be imported as vanilla css. This is breaking the styles.

Reply:
I also noted one additional build error complaining about an image it couldn't find:
```
Error downloading image: failed to process
https://dl.airtable.com/.attachments/1a6628a605e4c91142674c80321b3806/57ea260e/header_sharedhorizons.jpg
RequestError: getaddrinfo ENOTFOUND dl.airtable.com
```

Reply:
ah, good catch. Once upgraded, it didn't like to be imported as a module without an explicit export in the CSS. Let me look into it again